### PR TITLE
Update generate keys instructions

### PR DESCRIPTION
**Pull request by georgeweiler**
updates Mac, Windows & Linux instuctions to meet new requiments outlined in #138 

Screen Shot 2020-07-30 at 5 17 19 PM

### CODE REVIEW

**Comment by georgeweiler** [comment]
Hey @CarlBeek @hwwhww @Battenfield, the diff on this PR is a bit hard to follow because I extracted a lot of the copy out to `<Option1/>` and `<Option2>` components. I suggest that you please pull this branch down locally and check out the new generate keys page manually.

Happy to change any copy or styling.

File: src/pages/GenerateKeys/Option2.tsx [hunk 1]
```diff
@@ -0,0 +1,283 @@
...
+        </Text>
+      );
+    }
+  };
+
+  const renderDepositKeyCommand = () => {
+    if (os === 'mac' || os === 'linux') {
+      return (
+        <pre className="my0">
+          ./deposit{' '}
```
Comment (hwwhww) [inline src/pages/GenerateKeys/Option2.tsx:139]:
```suggestion
          python3 ./eth2deposit/deposit.py{' '}
```

File: src/pages/GenerateKeys/Option2.tsx [hunk 2]
```diff
@@ -0,0 +1,283 @@
...
+      <Alert variant="secondary" className="my10">
+        <pre className="my0">
+          <span style={{ color: colors.blue.medium }}>virtualenv</span> venv
+        </pre>
+        <pre className="my0">./venv/bin/activate</pre>
+      </Alert>
+      <Text>Second, install the dependency packages:</Text>
+      <Alert variant="secondary" className="my10">
+        <pre className="my0">
+          <span style={{ color: colors.red.medium }}>python3 setup</span>
```
Comment (hwwhww) [inline src/pages/GenerateKeys/Option2.tsx:237]: A forking here:

For Linux/macOS users, the same:
```
          <span style={{ color: colors.red.medium }}>python3 setup</span>
```

But for Windows users (assume `python` is python 3):

```
          <span style={{ color: colors.red.medium }}>python setup</span>
```

File: src/pages/GenerateKeys/Option2.tsx [hunk 3]
```diff
@@ -0,0 +1,283 @@
...
+          <span style={{ color: colors.purple.dark }}>.py</span>{' '}
```
Comment (hwwhww) [inline src/pages/GenerateKeys/Option2.tsx:238]: missing `install`
```suggestion
          <span style={{ color: colors.purple.dark }}>.py</span>{' '}<span style={{ color: colors.red.medium }}>install</span>
```

File: src/pages/GenerateKeys/Option1.tsx [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+          official GitHub account.
+        </Text>
+      </Alert>
+      <Text weight={500}>
+        Generate deposit keys using the Ethereum Foundation deposit tool
+      </Text>
+      <Text className="mt5">
+        Type the following lines into the terminal window:
+      </Text>
+      <Alert variant="secondary" className="my10">
```
Comment (Battenfield) [inline src/pages/GenerateKeys/Option1.tsx:48]: I'm unsure if we need this section. When I run the executable deposit from the download on OSX it automatically runs the script. @hwwhww @CarlBeek  should we remove this section? If so for which platforms?
Screen Shot 2020-07-31 at 5 10 06 PM

File: src/pages/GenerateKeys/Option1.tsx [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+        Option 1: Use binary executable file
+      </Heading>
+      <Text weight={500}>Download the binary executable file</Text>
+      <Text>
+        Please find the binary execution file of the latest deposit-cli release
+        on{' '}
+        <Link
+          primary
+          external
+          to="https://github.com/ethereum/eth2.0-deposit-cli/releases/"
```
Comment (Battenfield) [inline src/pages/GenerateKeys/Option1.tsx:29]: It might be unclear for users where the assets are to download on this Github Releases page. We may want to specify where the assets are located and which they should download for their OS. Alternatively, we could place the link to trigger the download upon click for their OS. If we go this route we would want to make sure the user is not being phished as they could be downloading a file from an untrusted source

**Comment by Battenfield** [comment]
@hwwhww @CarlBeek feel free to merge when you are ready, and then we can release to prod. I did leave a couple of comments on the executable instructions, but it could be something we clean up later.

File: src/pages/GenerateKeys/Option1.tsx [hunk 6]
```diff
@@ -0,0 +1,67 @@
...
+  os: string;
+}) => {
+  return (
+    <Paper className="mt20">
+      <Heading level={2} size="small" color="blueMedium" className="mb20">
+        Option 1: Use binary executable file
+      </Heading>
+      <Text weight={500}>Download the binary executable file</Text>
+      <Text>
+        Please find the binary execution file of the latest deposit-cli release
```
Comment (hwwhww) [inline src/pages/GenerateKeys/Option1.tsx:24]:
```suggestion
        Please find the binary executable file of the latest deposit-cli release
```

File: src/pages/GenerateKeys/Option1.tsx (hunk 4 above)
Comment (hwwhww) [reply src/pages/GenerateKeys/Option1.tsx:48]: Oh interesting. Did you see the "Please choose how many validators you wish to run: Please choose how many validators you wish to run:" automatically?

I'm on macOS, when I clicked the file it, it automatically runs with "deposit; exit" command. But I need to input a new line to get enter the prompt and see "Please choose how many validators you wish to run: Please choose how many validators you wish to run:". (on both bash or zsh)

image

**Comment by hwwhww** [comment]
@Battenfield  thanks for your review and trying it out! I'm merging it now and we can update the PR feedback with other PRs.
